src/api/types.ts

```diff
@@ -211,12 +211,41 @@ export type NFT = {
   image_url: string;
   /** URL of metadata */
   metadata_url: string;
-  /** Date of NFT creation */
-  created_at: string;
   /** Date of latest NFT update */
   updated_at: string;
   /** Whether NFT is disabled for trading on OpenSea */
   is_disabled: boolean;
   /** Whether NFT is NSFW (Not Safe For Work) */
   is_nsfw: boolean;
+  /** Traits for the NFT, returns null if the NFT has than 50 traits */
+  traits: Trait[] | null;
 };
+
+/**
+ * Trait type returned by OpenSea API.
+ * @category API Models
+ */
+export type Trait = {
+  /** The name of the trait category (e.g. 'Background') */
+  trait_type: string;
+  /** A field indicating how to display. None is used for string traits. */
+  display_type: TraitDisplayType;
+  /** Display type of trait */
+  max_value: string;
```

Comment:
just to confirm, is string correct here?

Reply:
yes but the comment is wrong, good catch. has to be string even if it's a number for big numbers or decimals
![Uploading Screenshot 2024-01-02 at 5.27.53 PM.png…]()